bin/tx-prover/README.md

```diff
@@ -0,0 +1,29 @@
+# miden-tx-prover-service
+
+A service that generates proofs on-demand.
+
+## Installation
+
+To install the prover service, run:
+
+```bash
+make install-prover
+```
```

Comment:
Either in addition to, or instead of this, should we describe how to install using `cargo install` (i.e., to pull directly from crates.io)?

Reply:
I added that using the miden-node docs as base, also I assume that the package is going to be called "miden-tx-prover". Let me know if you prefer any other name.

While doing this I tested the `cargo install` commands from the node and found that those are not working. I'm opening a PR there too.